python/sglang/srt/layers/moe/ep_moe/layer.py

```diff
@@ -109,15 +117,34 @@ def forward(
         if self.use_flashinfer:
             # TODO: flashinfer
             assert False
-            assert GroupedGemmRunner.flashinfer_gemm_warpper is not None
-            c = GroupedGemmRunner.flashinfer_gemm_warpper.run(
+            assert GroupedGemmRunner.flashinfer_gemm_wrapper is not None
+            c = GroupedGemmRunner.flashinfer_gemm_wrapper.run(
                 x=a,
                 weights=b,
                 batch_size=batch_size,
                 weight_column_major=weight_column_major,
                 seg_indptr=seg_indptr,
                 weight_indices=weight_indices,
             )
+        elif self.use_fbgemm and weight_column_major:
+            b_fbgemm = b.contiguous().reshape(-1, b.shape[2])
+            assert seg_indptr is not None, "FBGemm needs seg_indptr"
+            m_sizes = seg_indptr[1:] - seg_indptr[:-1]
```

Comment:
```suggestion
            m_sizes = (seg_indptr[1:] - seg_indptr[:-1]).to(torch.int32)
```

Change the line to avoid fbgemm kernel compile error.